docs/pipelines/add-pipeline.md

```diff
@@ -19,8 +19,11 @@ required_permissions:
 1. In the list of applications find the application where you want to add a pipeline and click `DETAILS`
 1. On the upper-right corner of the applications details view click `SETTINGS`
```

Comment:
I believe this step also needs to be updated, as I couldn't find a `SETTINGS` UI element to click at.